src/Admin/OptionsMenu.php

```diff
@@ -0,0 +1,487 @@
+			AMP_Options_Manager::OPTION_NAME
+		);
+
+		add_settings_section(
+			'validation',
+			false,
+			'__return_false',
+			AMP_Options_Manager::OPTION_NAME
+		);
+
+		add_settings_field(
+			Option::SUPPORTED_TEMPLATES,
+			__( 'Supported Templates', 'amp' ),
+			[ $this, 'render_supported_templates' ],
+			AMP_Options_Manager::OPTION_NAME,
+			'general',
+			[
+				'class' => 'amp-template-support-field',
+			]
+		);
+
+		/**
+		 * This fires when settings fields for the AMP Options menu need to be registered.
+		 *
+		 * This action is intended for internal use only, not to be used by other plugins.
+		 *
+		 * @internal
+		 */
+		do_action( 'amp_options_menu_items' );
+
+		$submenus = [
+			new AMP_Analytics_Options_Submenu( AMP_Options_Manager::OPTION_NAME ),
+		];
+
+		// Create submenu items and calls on the Submenu Page object to render the actual contents of the page.
+		foreach ( $submenus as $submenu ) {
+			$submenu->init();
+		}
+	}
+
+	/**
+	 * Provides the settings screen handle.
+	 *
+	 * @return string
+	 */
+	public function screen_handle() {
+		return sprintf( 'toplevel_page_%s', AMP_Options_Manager::OPTION_NAME );
+	}
+
+	/**
+	 * Enqueues settings page assets.
+	 *
+	 * @since 1.6.0
+	 *
+	 * @param string $hook_suffix The current admin page.
+	 */
+	public function enqueue_assets( $hook_suffix ) {
+		if ( $this->screen_handle() !== $hook_suffix ) {
+			return;
+		}
+
+		$asset_file   = AMP__DIR__ . '/assets/js/' . self::ASSET_HANDLE . '.asset.php';
+		$asset        = require $asset_file;
+		$dependencies = $asset['dependencies'];
+		$version      = $asset['version'];
+
+		wp_enqueue_script(
+			self::ASSET_HANDLE,
+			amp_get_asset_url( 'js/' . self::ASSET_HANDLE . '.js' ),
+			$dependencies,
+			$version,
+			true
+		);
+
+		wp_enqueue_style(
+			self::ASSET_HANDLE,
+			amp_get_asset_url( 'css/amp-settings.css' ),
+			[ $this->google_fonts->get_handle(), 'wp-components' ],
+			AMP__VERSION
+		);
+
+		wp_styles()->add_data( self::ASSET_HANDLE, 'rtl', 'replace' );
+
+		$theme           = wp_get_theme();
+		$is_reader_theme = in_array( get_stylesheet(), wp_list_pluck( ( new AMP_Reader_Themes() )->get_themes(), 'slug' ), true );
+
+		$js_data = [
+			'BUILT_IN_SUPPORT'                   => in_array( get_template(), AMP_Core_Theme_Sanitizer::get_supported_themes(), true ),
+			'CURRENT_THEME'                      => [
+				'name'            => $theme->get( 'Name' ),
+				'description'     => $theme->get( 'Description' ),
+				'is_reader_theme' => $is_reader_theme,
+				'screenshot'      => $theme->get_screenshot(),
+				'url'             => $theme->get( 'ThemeURI' ),
+			],
+			'OPTIONS_REST_ENDPOINT'              => rest_url( 'amp/v1/options' ),
+			'READER_THEMES_REST_ENDPOINT'        => rest_url( 'amp/v1/reader-themes' ),
+			'THEME_SUPPORT_ARGS'                 => AMP_Theme_Support::get_theme_support_args(),
+			'THEME_PROVIDED_SUPPORT_MODE'        => AMP_Theme_Support::get_support_mode_added_via_theme(),
```

Comment:
@westonruter Can you advise on how best to replace this `AMP_Theme_Support::get_support_mode_added_via_theme()`? On the JS side it's expecting `THEME_PROVIDED_SUPPORT_MODE` to be 'reader', 'standard', or 'transitional'.

Reply:
Instead of `AMP_Theme_Support::get_support_mode_added_via_theme()` you should be able to just use `current_theme_supports('amp')`

Reply:
The `amp` theme support is now only added/removed when actually serving an AMP page:

https://github.com/ampproject/amp-wp/blob/af10876fdf076cc4ecd4d6a9c58f8eba308302f4/includes/class-amp-theme-support.php#L332-L352

Reply:
OK, I think I need to write a new method to make this work. It determines the recommendation notices showing on the template modes on the settings screen.

Reply:
Actually, if the current theme support AMP then we can suggest _either_ `standard` _or_ `transitional`. The `get_support_mode_added_via_theme` is deprecated now because it is obsolete. The reality is that if a theme supports AMP then it should work in either mode.

Otherwise, if we really want to be _traditional_ we could just use `THEME_SUPPORT_ARGS` and if `!! THEME_SUPPORT_ARGS.paired` then the theme-provided mode is `transitional` and otherwise it is `standard`.